### PR TITLE
docs: add videos to verification doc

### PR DESCRIPTION
**Pull request by arjunattam**
Ref #3996

### CODE REVIEW

File: docs/verification.md [hunk 1]
```diff
@@ -9,14 +10,38 @@
 
 <br/>
 
+## Videos
+
+Playwright can record videos for all pages in a [browser context](core-concepts.md#browser-contexts).
+
+```js
+// With browser.newContext
```
Comment (mxschmitt) [inline docs/verification.md:18]: nit, demonstrates that its a function
```suggestion
// With browser.newContext()
```

File: docs/verification.md [hunk 2]
```diff
@@ -9,14 +10,38 @@
...
+const context = await browser.newContext({ videosPath: 'videos/' });
+
+// With browser.newPage
```
Comment (mxschmitt) [inline docs/verification.md:21]:
```suggestion
// With browser.newPage()
```

File: docs/verification.md (hunk 1 above)
Comment (mxschmitt) [inline docs/verification.md:18]: nit: demonstrates that its a function
```suggestion
// With browser.newContext()
```

**Review by mxschmitt — APPROVED** [review]

File: docs/verification.md [hunk 3]
```diff
@@ -9,14 +10,38 @@
...
+const page = await browser.newPage({ videosPath: 'videos/' });
+
+// [Optional] Specify video size; defaults to viewport size
+const context = await browser.newContext({
+    videosPath: 'videos/',
+    videoSize: { width: 1920, height: 1080 }
```
Comment (dgozman) [inline docs/verification.md:27]: nit: I'd go for something smaller here 😄

**Review by dgozman — APPROVED** [review]

**Pull request merged**